skype/skype-ps/skype/New-CsTeamsCallParkPolicy.md

```diff
@@ -102,6 +115,55 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -PickupRangeEnd
+Specify the maximum value that a rendered pickup code can take. Value can be from 10 to 9999.
+
+Note: PickupRangeStart must be smaller than PickupRangeEnd.
+
+```yaml
+Type: Integer
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: 99
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ParkTimeoutSeconds
+Specify the number of seconds to wait before ringing the parker when the parked call hasn't been picked up. Value can be from 120 to 1800 (seconds).
```

Comment:
```suggestion
Specify the number of seconds to wait before ringing the parker when the parked call hasn't been picked up. Value can be from 60 to 1800 (seconds).
```

![image](https://user-images.githubusercontent.com/33589238/136232590-eb7adcb6-426a-4c2a-b78d-bd99562d82e9.png)

Reply:
Hi, the changes updating this value to 120 is currently rolling out and is available in lower rings.
![image](https://user-images.githubusercontent.com/17123411/136257416-d58e1051-3009-4f96-81aa-c0654eac38ce.png)
Once it's available in all rings, I'll update this thread.